book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Kindly set getters and setters method for rentals by adding it to attr_accessor.

Reply:
according to the requirements we have to put rentals just on reader mode

Reply:
Yes you don't need that. We would like to apologize for the inconvenience